packages/terra-functional-testing/src/config/utils/getConfigurationOptions.js

```diff
@@ -10,6 +10,7 @@ const getConfigurationOptions = (options) => {
   const {
     assetServerPort,
     browsers,
+    buildBranch,
```

Comment:
Can we update [getConfigurationOptions.test.js](https://github.com/cerner/terra-toolkit/blob/main/packages/terra-functional-testing/tests/jest/config/utils/getConfigurationOptions.test.js) to test this option? It should be similar to this [change](https://github.com/cerner/terra-toolkit/pull/732/files#diff-8068d3d73720a3af093924b9be59a1832d5e1ab4062cea3ce95310329b1159bb).

Reply:
Added to the tests here https://github.com/cerner/terra-toolkit/pull/736/commits/dd9fbb2487069446ec10a8e4109fd6729ab9599e#diff-8068d3d73720a3af093924b9be59a1832d5e1ab4062cea3ce95310329b1159bbR12